### PR TITLE
allow match  accommodations with same providers

### PR DESCRIPTION
**Pull request by hasmikharutyunyan**

### CODE REVIEW

**Review by DFSko — APPROVED** [review]

File: HappyTravel.Nakijin.Api/Services/AccommodationManagementService.cs [hunk 1]
```diff
@@ -297,39 +294,39 @@ public async Task<Result> Activate(int accommodationId, CancellationToken cancel
...
 
         public async Task<Result> Activate(ActivationRequest request, CancellationToken cancellationToken = default)
         {
             if (!request.AccommodationIds.Any())
                 return Result.Failure("Accommodation ids must be specified");
-            
-            var accommodationsToActivate = await _context.Accommodations.Where(a => request.AccommodationIds.Contains(a.Id) && !a.IsActive).ToListAsync(cancellationToken);
+
+            var accommodationsToActivate = await _context.Accommodations.Where(a => request.AccommodationIds.Contains(a.Id) && !a.IsActive)
+                .ToListAsync(cancellationToken);
```
Comment (mingazhev) [inline HappyTravel.Nakijin.Api/Services/AccommodationManagementService.cs:322]: Will be more readable with the blank like below

Comment (mingazhev) [inline HappyTravel.Nakijin.Api/Services/AccommodationManagementService.cs:322]: Will be more readable with the blank like below

File: HappyTravel.Nakijin.Api/Services/AccommodationManagementService.cs [hunk 2]
```diff
@@ -369,41 +368,41 @@ public async Task<Result> Deactivate(int accommodationId, DeactivationReasons re
...
-                
+
                 return accommodation;
             }
 
-            
-            Task Unpublish(RichAccommodationDetails accommodation)
-                => _accommodationChangePublisher.PublishRemoved(accommodation.Id);
+
+            Task Unpublish(RichAccommodationDetails accommodation) => _accommodationChangePublisher.PublishRemoved(accommodation.Id);
```
Comment (mingazhev) [inline HappyTravel.Nakijin.Api/Services/AccommodationManagementService.cs:385]: Arrow to the next line

File: HappyTravel.Nakijin.Api/Services/AccommodationManagementService.cs [hunk 3]
```diff
@@ -217,13 +217,10 @@ private async Task<Result> Match(int sourceHtId, int htIdToMatch)
 
             if (sourceAccommodation == default || accommodationToMatch == default)
                 return Result.Failure("Wrong accommodation Id");
-
+            
             foreach (var supplierAccommodation in accommodationToMatch.SupplierAccommodationCodes)
```
Comment (mingazhev) [inline HappyTravel.Nakijin.Api/Services/AccommodationManagementService.cs:221]: Should not we log such things, at least on warning level?

**Pull request closed without merging**